x11-themes/chameleon-xcursors/chameleon-xcursors-0.5-r1.ebuild

```diff
@@ -1,7 +1,8 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
+inherit xdg-utils
```

Comment:
Why not use `xdg.eclass` and remove the `pkg_` phases?

Reply:
:open_mouth: 
I was not aware of this one, thanks a lot!
Gonna update them.